pkg/api/volume_req.go

```diff
@@ -2,20 +2,16 @@ package api
 
 // BrickReq represents Brick Request
 type BrickReq struct {
-	Type           string `json:"type"`
-	PeerID         string `json:"peerid"`
-	Path           string `json:"path"`
-	TpMetadataSize uint64 `json:"metadata-size,omitempty"`
-	TpSize         uint64 `json:"thinpool-size,omitempty"`
-	VgName         string `json:"vg-name,omitempty"`
-	TpName         string `json:"thinpool-name,omitempty"`
-	LvName         string `json:"logical-volume,omitempty"`
-	Size           uint64 `json:"size,omitempty"`
-	VgID           string `json:"vg-id,omitempty"`
-	Mountdir       string `json:"mount-dir,omitempty"`
-	DevicePath     string `json:"device-path,omitempty"`
-	MntOpts        string `json:"mnt-opts,omitempty"`
-	FsType         string `json:"fs-type,omitempty"`
+	Type       string `json:"type"`
+	PeerID     string `json:"peerid"`
+	Path       string `json:"path"`
+	Name       string `json:"name,omitempty"`
+	Size       uint64 `json:"size,omitempty"`
+	Mountdir   string `json:"mount-dir,omitempty"`
+	Device     string `json:"device,omitempty"`
```

Comment:
I need to think about different naming for this. Device is main device where VG is created in case of lvm. DevicePath is path of LV where brick needs to be mounted. Is `RootDevice` or `ParentDevice` sounds good instead of just `Device`?